library/common/main_interface.h

```diff
@@ -25,9 +25,11 @@ envoy_stream_t init_stream(envoy_engine_t engine);
  * can occur.
  * @param stream, handle to the stream to be started.
  * @param callbacks, the callbacks that will run the stream callbacks.
+ * @param explicit_flow_control, whether request/response buffers will be managed explicitly.
```

Comment:
along the line of Alyssa's comment, lets make it clear everywhere that right now this is only wired for the response path, but request path is coming.

Although, that being said I am confused what the request path means for this type of flow control if the client is the one sending request bytes down to envoy mobile with the send* API, so it is _always_ explicitly managed by the client.

Reply:
I believe most (all?) of my docstring comments call out the response path specifically. Alyssa's PR implementing request path is posted.

Reply:
Updated this comment, which was the only one that I think mentioned request buffers.